app/controllers/had_skills_controller.rb

```diff
@@ -0,0 +1,75 @@
+class HadSkillsController < ApplicationController
+  before_action :authenticate_user!, only: [:new, :create, :update, :destroy]
+  before_action :had_skill, only: [:show, :edit, :update]
+  before_action :get_user, only: [:new, :create]
+
+  # move to model?
+  def titleize(string)
+    lowercase_words = %w{a an the and but or for nor of}
+    string.split.each_with_index.map{|x, index| lowercase_words.include?(x) && index > 0 ? x : x.capitalize }.join(" ")
```

Comment:
Line is too long. [119/80]<br>Space missing to the left of {.<br>Space between { and | missing.